ServiceFabric.ServiceBus.Services.Netstd/CommunicationListeners/ServiceBusQueueCommunicationListener.cs

```diff
@@ -21,27 +21,27 @@ public class ServiceBusQueueCommunicationListener : ServiceBusQueueCommunication
         /// <summary>
         /// Processor for messages.
         /// </summary>
-        protected IServiceBusMessageReceiver Receiver { get; }
+        protected IServiceBusMessageHandler Handler { get; }

```

Comment:
These are also breaking changes, please revert back.

Reply:
Some people can be very eager 🤣